tests/dialects/test_snowflake.py

```diff
@@ -1282,6 +1282,13 @@ def test_ddl(self):
             read={"teradata": "CREATE MULTISET TABLE a (b INT)"},
             write={"snowflake": "CREATE TABLE a (b INT)"},
         )
+        self.validate_all(
+            """
+                ALTER TABLE a
+                ALTER COLUMN my_column DROP NOT NULL;
+            """,
+            write={"duckdb": "ALTER TABLE a ALTER COLUMN my_column DROP NOT NULL"},
+        )
```

Comment:
Make sure to always test the roundtrip, i.e. snowflake -> snowflake. So we need a couple of entries here.